aquacomputer_d5next.c

```diff
@@ -211,6 +218,29 @@ static const char *const label_fan_flow_speed_quadro[] = {
 	"Flow speed [l/h]"
 };
 
+struct fan_structure_offsets {
+	u8 voltage;
+	u8 ampere;
+	u8 power;
+	u8 speed;
+};
+
+/* Fan structure offsets for Aquaero */
+static struct fan_structure_offsets fan_structure_aquaero = {
+	.voltage = AQUAERO_FAN_VOLTAGE_OFFSET,
+	.ampere = AQUAERO_FAN_CURRENT_OFFSET,
+	.power = AQUAERO_FAN_POWER_OFFSET,
+	.speed = AQUAERO_FAN_SPEED_OFFSET
+};
+
+/* Fan structure offsets for D5 Next, Octo and Quadro */
```

Comment:
Not only those, but all others except the Aquaero. I'd reword this to convey that Aquaero is the exception.